src/layer/form-widgets/index.md

```diff
@@ -0,0 +1,298 @@
+
+Extra configuration of attribute forms can be done to make data collection easier and more consistent, such as using default values, conditional visibility or constraint enforcement. These topics are covered in [Attributes Form Configuration](../form-configuration/) and [Attributes Form Layout](../form-layout/).
+
+<YouTube id="jc4p1PpXj3k" />
+
+## Widget gallery
+Attributes forms can be configured using <QGISHelp ver="latest" link="user_manual/working_with_vector/vector_properties.html#edit-widgets" text="QGIS widget types" /> in the **Attributes Form** tab in the **Layer Properties**.
+
+A number of QGIS widget types can be used in <MobileAppName />, such as *Text Edit*, *Range* widget for numbers, *Date/Time* widget with calendar popup, *Value Map* and *Value Relation* to select options from a drop-down menu, or the *Attachment* widget that is used to save photos. The widget has to be compatible with the data type of a field. 
+
+Here is an overview of widgets commonly used both in <QGIS link="" text="QGIS" /> and the <MobileAppNameShort />.
+
+|QGIS widget  | Description  |<div style="width:300px">Preview in <MobileAppName /> </div> |
+|:---:|:---:|:---:|
+|Text Edit  |[Text](#text)   | ![Mergin Maps mobile app text field form](./mobile-form-text-preview.jpg "Mergin Maps mobile app text field form") |
+|QR & barcode scanner  |[Camera to scan QR and barcode](#qr-code)   | ![Mergin Maps mobile app QR code field form](./mobile-form-qrcode-preview.jpg "Mergin Maps mobile app QR code field form") |
+|Range - Editable   |[Enter a number](#numbers)   | ![Mergin Maps mobile app numeric field form](./mobile-form-number-editable-preview.jpg "Mergin Maps mobile app numeric field form")  |
+|Range - Slider  |[Select a number using the slider](#slider)   | ![Mergin Maps mobile app slider field form](./mobile-form-number-slider-preview.jpg "Mergin Maps mobile app slider field form")  |
+|Date/Time   |[Calendar with time](#date-and-time)  |![Mergin Maps mobile app date time field form](./mobile-form-datetime-preview.jpg "Mergin Maps mobile app date time field form")   | 
+|Checkbox   |[Checkbox](#checkbox)   |![Mergin Maps mobile app checkbox field form](./mobile-form-checkbox-preview.jpg "Mergin Maps mobile app checkbox field form")   |
+|Value Map   |[Drop-down menu with predefined values](#value-map) |![Mergin Maps mobile app value map field form](./mobile-form-valuemap-preview.jpg "Mergin Maps mobile app value map field form")   |
+|Value Relation   |[Drop-down menu with values from another table](#value-relation) |![Mergin Maps mobile app value relation field form](./mobile-form-value-relation-preview.jpg "Mergin Maps mobile app value relation field form")   |
+|Attachment   |[Photos from device's camera or gallery](#attachment) | ![Mergin Maps mobile app attachment field form](./mobile-form-photo-preview.jpg "Mergin Maps mobile app attachment field form")  |
```

Comment:
Shall we show an actual picture here instead of the empty state of the widget? What do you think?

Reply:
added both empty widget and actual picture. what do you think?

Reply:
Nice! :) Thanks